### PR TITLE
fix: rationalize CSS

### PR DESCRIPTION
**Pull request by jessicaschilling**
Ref https://github.com/protocol/nft-website/issues/5

This PR:
- Fixes the `!important` declarations made earlier when applying branding (note: there are still two instances inherited from the original IPFS docs codebase)
- Moves CSS that should be scoped into the individual components in `/docs/.vuepress/theme/components` and `/docs/.vuepress/theme/global-components`
- Tidies CSS overall

@zebateira, I think this is ready for your expert review and any additional CSS tidying you can find. 😊  Hopefully there will be less cleanup required than you anticipate! Thanks much.

### CODE REVIEW

**Comment by filecorgi** [comment]
- I optimized some images for you! See the commit with the comment `Automatically optimized images [ci skip].` in this PR for details.
- Vuepress build was successful!

File: docs/.vuepress/theme/styles/header.styl [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+	border-bottom: none;
+
+	.logo {
+		height: 2.5rem;
+		margin: 0;
+		padding: 1px 3px 0px 0;
+		vertical-align: top;
+	}
+
+		.site-name {
```
Comment (zebateira) [inline docs/.vuepress/theme/styles/header.styl:14]:
```suggestion
	.site-name {
```

**Review by zebateira — APPROVED** [review]
LGTM 👏 
I made some structural tweaks and a ui fix on the search input (let me know if it makes sense).

**Comment by jessicaschilling** [comment]
Thanks much! Search box approach makes sense. I'll migrate it to the non-Algolia version in https://github.com/protocol/nft-website/pull/62 too. 🤣

**Pull request merged**